api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/standalone/ExampleService.java

```
@@ -146,14 +146,14 @@ private void generateExample(String serviceId, OpenAPI swagger, String method, O
     public void generateExamples(String serviceId, String apiDoc) {
         try {
             SwaggerParseResult parseResult = new OpenAPIParser().readContents(apiDoc, null, null);
-
-            OpenAPI swagger = parseResult.getOpenAPI();
-            Paths paths = swagger.getPaths();
-
+            OpenAPI swagger = parseResult != null ? parseResult.getOpenAPI() : null;
+            Paths paths = swagger != null ? swagger.getPaths() : null ;
+            if (paths != null) {
```

if `paths` is null it should throw an exception